homeassistant/components/webhook/trigger.py

```diff
@@ -62,6 +85,32 @@ async def async_attach_trigger(
 ) -> CALLBACK_TYPE:
     """Trigger based on incoming webhooks."""
     webhook_id: str = config[CONF_WEBHOOK_ID]
+    local_only = config.get(CONF_LOCAL_ONLY)
+    issue_id: str | None = None
+    if local_only is None:
+        issue_id = f"trigger_missing_local_only_{webhook_id}"
+        variables = trigger_info["variables"] or {}
+        automation_info = variables.get("this", {})
+        automation_id = automation_info.get("attributes", {}).get("id")
+        automation_entity_id = automation_info.get("entity_id")
+        automation_name = trigger_info.get("name") or automation_entity_id
```

Comment:
This is not really pretty, but does the job. I don't see a better or more elegant solution at this point either.

Considering the importance and temporary nature of this code (during deprecation) I think this is acceptable/OK for now.